Post release improvements-v2

JavaDogWebDesign opened the pull request:
Enhancements/improvements:

- mobile menu click out
- detect system theme
- linking images properly
- minifying css / js (not less)

JavaDogWebDesign committed:
css, favicon, cleaned tags

JavaDogWebDesign committed:
moved critical to local

JavaDogWebDesign committed:
click bg to click out of mobile menu & minify js

JavaDogWebDesign committed:
css change path

JavaDogWebDesign committed:
added critical

JavaDogWebDesign committed:
bg added mobile menu

JavaDogWebDesign committed:
clean up menu

JavaDogWebDesign committed:
added css and fixed right photo

JavaDogWebDesign committed:
css and canonical updates

JavaDogWebDesign committed:
parmalink for blog posts

JavaDogWebDesign committed:
edited text

JavaDogWebDesign committed:
css link fixed

JavaDogWebDesign committed:
minify css

JavaDogWebDesign committed:
img link fix

JavaDogWebDesign committed:
Koala preprocessor minifying process

JavaDogWebDesign committed:
fixed file path to display img

JavaDogWebDesign committed:
adding To the Top btn

monkeywhovapes commented:
Notable enhancements are:

- mobile menu "click out" of the menu
- moved critial to local, no need for multiple small css files when its a very small set of code
- changing favicon based on system theme preference
- changing site theme based on system theme preference
- minifying process with koala
- added "to the top" button for accessibility requirements
- general clean up / proper linking of files

JavaDogWebDesign commented:
Notable enhancements are:

- mobile menu "click out" of the menu
- moved critial to local, no need for multiple small css files when its a very small set of code
- changing favicon based on system theme preference
- changing site theme based on system theme preference
- minifying process with koala
- added "to the top" button for accessibility requirements
- general clean up / proper linking of files

JavaDogWebDesign committed:
source local files

JavaDogWebDesign committed:
improved looks for totopbtn
